bin/import.ml

```diff
@@ -1,38 +1,50 @@
 open Stdune
-open Dune_engine
+
+include struct
+  open Dune_engine
+  module Build_config = Build_config
+  module Build_system = Build_system
+  module Load_rules = Load_rules
+  module Package = Package
+  module Hooks = Hooks
+  module Action_builder = Action_builder
+  module Action = Action
+  module Dep = Dep
+  module Action_to_sh = Action_to_sh
+  module Dpath = Dpath
+  module Findlib = Dune_rules.Findlib
+  module Dune_package = Dune_rules.Dune_package
+  module Install = Dune_rules.Install
+  module Section = Section
+  module Diff_promotion = Diff_promotion
+  module Dune_project = Dune_project
+  module Cached_digest = Cached_digest
+  module Targets = Targets
+end
+
+include struct
+  open Dune_rules
+  module Super_context = Super_context
+  module Context = Context
+  module Config = Dune_util.Config
+  module Lib_name = Lib_name
+  module Colors = Colors
```

Comment:
Hmm is this supposed to be here?

Reply:
It was only used once so I moved it to the call site.